extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -138,7 +138,32 @@ function renderActiveFeedback(video, response) {
     };
 }
 
-function renderPassiveFeedback(video, response) {
+function displayContextKeyWords(label, keywordsMap) {
```

Comment:
Please do me a favor and declare the `keywordsMap` on the top of the file

`let keywordsMap = new Map()`

for things like the `event` object which dynamic every call, we should surely pass it as parameter. I hardly see any reason for something that is likely to be invariant throughout the lifecycle after the initial definition

Global variables are encouraged and favored over passing variables.